x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/outline_logo.ts

```diff
@@ -0,0 +1,50 @@
+import { css, html, LitElement, nothing } from "lit";
```

Comment:
@daniellacosse that was the first approach I tried but I could not figure out a way to serve static content as the app could not find `./outline_logo.svg` file location. Do you have any tips on serving static content in `Index.ts`?

Reply:
The reason I opted for creating a custom Lit element for the logo was to avoid framework specific setup. The current approach only works for SVG or text-based assets though. It seems like we don't need to do that for CSS assets here since they are encapsulated in custom Lite elements in the code.

For PNG/JPG/etc images, we need to offer a way to serve static assets though. 

If you agree, I can add this as `TODO` item as technical debt / improvement to `README` and do it via a separate PR.